lib/workers/repository/updates/generate.ts

```diff
@@ -206,7 +207,14 @@ export function generateBranchConfig(
     if (upgrade.toLowerCase) {
       // We only need to lowercase the first line
       const splitMessage = upgrade.commitMessage.split(newlineRegex);
-      splitMessage[0] = splitMessage[0].toLowerCase();
+      // Do not lowercase semantic prefix
+      if (upgrade.useSemanticCommit && upgrade.commitMessagePrefix) {
```

Comment:
@rarkins this is even more hacki and not really maintainable

Reply:
It definitely is, but I fear that without a major refactoring and overhaul of the whole rendering of the commitMessage there is no way to fix the issue. IMHO it would also be fixed by removing https://github.com/renovatebot/renovate/blob/main/lib/workers/repository/updates/generate.ts#L181-L183 altogether with the disadvantage of potentially breaking processes for people who rely on the lowercased subject.